src/core/logger.js

```diff
@@ -9,10 +9,11 @@
 const DEV =
```

Comment:
Are you able to see any log messages on the browser? Unless someone uses a Webpack plugin (or equivalent) to set `process.env.NODE_ENV` here I would only expect these log messages to show up on Node.

This is not directly related to this PR, but I thought I'd ask in case you add this new log message and don't see the output.